web-client/src/App.test.tsx

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
It is recommended to add tests for the existing code.

Reply:
I am going to add tests later, after learning with this topic